strategy/live_exec.py

```
@@ -478,26 +792,214 @@ def redeem(condition_id: str, index_sets: list[int] | None = None,
+            if up_tok_id:
+                r_up = c.get_balance_allowance(
+                    BalanceAllowanceParams(asset_type=AssetType.CONDITIONAL, token_id=up_tok_id, signature_type=sig_type)
+                )
+                up_bal = float(r_up.get("balance", 0) or 0) / 1e6
+            if dn_tok_id:
+                r_dn = c.get_balance_allowance(
+                    BalanceAllowanceParams(asset_type=AssetType.CONDITIONAL, token_id=dn_tok_id, signature_type=sig_type)
+                )
+                dn_bal = float(r_dn.get("balance", 0) or 0) / 1e6
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)